store/sqlstore/post_store.go

```diff
@@ -1202,3 +1227,57 @@ func (s SqlPostStore) GetOldest() store.StoreChannel {
 		result.Data = &post
 	})
 }
+
+// GetMaxPostSize returns the maximum number of runes that may be stored in a post.
+func (s SqlPostStore) GetMaxPostSize(allowFromCache bool) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		var maxPostSize int32 = model.POST_MESSAGE_MAX_RUNES_V1
```

Comment:
I think it would be more idiomatic and simplify things to use `int` here and do a cast in the MySQL case.